src/navigationDrawer/navigationDrawer.component.tsx

```diff
@@ -130,7 +135,7 @@ export const NavigationDrawer = (
           )}
       </Fragment>
     );
-  };
+  }, [buildMenuSection, props]);
```

Comment:
This callback will be rebuilt when any of the props change - can we deconstruct the props needed i.e. `props.plugins` & `props.homepageUrl`

Reply:
Just fixed, I just used the recommended ones and it was fooled by destructuring assignment of plugins.